content/docs/user-guide/what-is-dvc.md

```diff
@@ -47,3 +47,12 @@ can version experiments, manage large datasets, and make projects reproducible.
 
   > Git servers, as well as SSH and cloud storage providers are supported,
   > however.
+
+## DVC is not Git!
+
+DVC metafiles such as `dvc.yaml` and `.dvc` files serve as placeholders to track
+large data files and directories for versioning (among other purposes). They
+provide the ability to store multiple data versions out-of-the-box, but this
+does not replace any SCM features. Full-fledged
```

Comment:
SCM -> Git

`They provide the ability to store multiple data versions out-of-the-box` - sound like a single file is responsible to multiple versions. (vs multiple versions of the same file)

I would try to rephrase (simplify this)- it's a bit confusing, is Git still does versioning? etc...

Reply:
True. I rewrote this (added a 2nd p too) in f447b143. PTAL